Shared/Supporting Files/be.lproj/Settings.strings

```diff
@@ -4,9 +4,6 @@
 /* Title label for when there are no credit cards shown in credit card list in autofill settings screen. %@ is the product name and should not be altered. */
 "CreditCard.Settings.EmptyListTitle.v112" = "Захаваць крэдытныя карты ў %@";
 
-/* Accessibility label for a credit card list item in autofill settings screen. The first parameter is the credit card type (e.g. Visa). The second parameter is the last 4 digits of the credit card. The third parameter is the name of the credit card holder. The fourth and fifth parameters are the month and year of the credit card's expiration date. */
-"CreditCard.Settings.ListItemA11y.v112" = "%1$@, якая заканчваецца на %2$@, выдадзена %3$@, тэрмін дзеяння %4$@";
```

Comment:
That string is still used in v117

Reply:
I think we could backport https://github.com/mozilla-mobile/firefox-ios/pull/15783 to fix this

Reply:
Backported @lmarceau